spectrum/xpub.py

```diff
@@ -10,6 +10,7 @@ class PageObject:
     def __init__(self, driver):
         self._driver = driver
 
+    # TODO: transform into XpubInput object?
```

Comment:
tried doing this, but at the moment doesn't add anything in clarity or decoupling